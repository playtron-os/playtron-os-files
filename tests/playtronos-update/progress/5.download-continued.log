Pulling manifest: ostree-unverified-registry:registry.playtron.one/internal/playtronos:0.19.0.20
Importing: ostree-unverified-registry:registry.playtron.one/internal/playtronos:0.19.0.20 (digest: sha256:5b3e1bdbe4481b15e2b79eb676822e0235103154ddd060c4cb16208427151747)
ostree chunk layers already present: 65
custom layers already present: 12
custom layers needed: 9 (872.9 MB)
Fetching layer sha256:96bfb5f0e88f (771.6 MB)...done
Fetching layer sha256:a3e87a34fed9 (2.1 kB)...done
Fetching layer sha256:c6bb52456661 (249 bytes)...done
Fetching layer sha256:dd219d91a113 (436 bytes)...done
Fetching layer sha256:6e41db4767dc (370 bytes)...done
Fetching layer sha256:a652d6d72151 (5.3 kB)...done
Fetching layer sha256:808f27ab08c0 (1.4 kB)...done
Fetching layer sha256:f3136b4d0468 (101.4 MB)...done
Fetching layer sha256:2451925bcc99 (1.3 kB)...done
Staging deployment...done
Upgraded:
  ModemManager 1.22.0-3.fc40 -> 1.22.0-4.fc41
  kernel-srpm-macros-1.0-24.fc41.noarch
  zix-0.4.2-2.fc41.x86_64
  zlib-ng-compat-devel-2.1.7-3.fc41.x86_64
Changes queued for next boot. Run "systemctl reboot" to start a reboot
