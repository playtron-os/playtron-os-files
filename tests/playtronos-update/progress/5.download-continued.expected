{ "done_mb": 771, "total_mb": 872, "done_parts": 1, "total_parts": 9 }
{ "done_mb": 771, "total_mb": 872, "done_parts": 2, "total_parts": 9 }
{ "done_mb": 771, "total_mb": 872, "done_parts": 3, "total_parts": 9 }
{ "done_mb": 771, "total_mb": 872, "done_parts": 4, "total_parts": 9 }
{ "done_mb": 771, "total_mb": 872, "done_parts": 5, "total_parts": 9 }
{ "done_mb": 771, "total_mb": 872, "done_parts": 6, "total_parts": 9 }
{ "done_mb": 771, "total_mb": 872, "done_parts": 7, "total_parts": 9 }
{ "done_mb": 873, "total_mb": 872, "done_parts": 8, "total_parts": 9 }
{ "done_mb": 873, "total_mb": 872, "done_parts": 9, "total_parts": 9 }
